examples/policies.py

```diff
@@ -83,7 +83,23 @@
 print("\n\ncreate policy")
 policy_name = "fixed_frequency_retention_policy"
 policy = policies.create(policy_name)
-print(policy, policy.data)
+print(f"{policy}")
+print(f"{pp.pformat(policy.data)} \n")
+rules = [
+    {
+        "start_time": "14:30",
```

Comment:
would many people have a start_time without a stated end_time

Reply:
Added the end_time parameter along with start_time parameter